tests/test_psf.py

```diff
@@ -372,6 +376,7 @@ def test_atm_psf_fft(self):
                     'rawSeeing': self.opsim_data['rawSeeing'],
                     'band':  self.opsim_data['band'],
                     'screen_size': 409.6,
+                    'nproc': 1,
```

Comment:
Were these causing problems using multiple processes?  I'm not sure why you added nproc: 1 to these.  I think it's a good idea to have at least one test use it, just to make sure things work with that parameter.  But similarly, we should keep at least one tests without it to make sure things work with the default nproc: None.  I'd even suggest adding one that is explicitly nproc: 6 to make sure that works ok. (Since that's what None normally turns into.)

Reply:
Yes, for some reason, I kept seeing the CI killed during test_psf.py, i.e., [here](https://github.com/LSSTDESC/imSim/actions/runs/6175279590/job/16762744913) and[ here](https://github.com/LSSTDESC/imSim/actions/runs/6176046866/job/16764176869).  I have no idea why it started to fail with my build, but I noted in the [github runner docs](https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources)  that linux VMs only have 7GB of RAM, so I made `nproc=1` to avoid the RSS exceeding that number.

Reply:
In that case, maybe just add that for the one that uses `'screen_size': 409.6,`  That's the only one that should have a large memory footprint.  (Although I don't understand why using multiple processes to calculate it would be connected with higher memory...)

Reply:
Oh, I guess there are temporary Fourier planes during the initialization.  So with nproc=6, you have all 6 of those in memory at once, rather than one at a time.  That does make sense.

Reply:
I still think this is the only one of these that should have `nproc: 1`.  The others should not have a significant memory footprint.